src/main/kotlin/gropius/model/common/SyncNode.kt

```diff
@@ -0,0 +1,51 @@
+package gropius.model.common
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.generator.annotations.GraphQLIgnore
+import gropius.model.user.User
+import io.github.graphglue.model.*
+import org.springframework.data.annotation.Transient
+import java.time.OffsetDateTime
+
+@DomainNode
+@GraphQLDescription(
+    """ExtensibleNode which provides information necessary for sync
+    When it was created and last modified, if the it is already deleted, and by who it was created and last modiied.
+    """
+)
+abstract class SyncNode(
```

Comment:
Unless we reabolish the sync service databases, I hate that name. I suggest something like `TimelinedNode`

Reply:
The problem is, these are not only TimelineItems (but also stuff like Issues, Labels, ...).  
But may be `AuditedNode` an alternative? As this process is in general called Auditing (https://docs.spring.io/spring-data/neo4j/docs/current/reference/html/#auditing).
Although we do not use the Spring features there

Reply:
Why change that name? 

I find `SyncNode` pretty self explanatory or what do you want to imply with the name @chriku

Reply:
renamed with c6c9d006cc402976123712e4494dc742d467848f